composer.json

```diff
@@ -24,7 +24,7 @@
     },
     "require": {
         "php": ">=5.5",
-        "symfony/symfony": "^2.7",
+        "symfony/symfony": "2.7.*",
```

Comment:
I didn't dig into it for now. PHPUnit get 500 errors on web tests, not with 2.7.

I would like to fix deprecated stuff before, then try to look at 2.8.